nyumbaAI/nyumbaAI_app/views.py

```
@@ -0,0 +1,75 @@
+from django.shortcuts import render, redirect
+from django.http import HttpResponseBadRequest
+from .models import SearchQuery, HouseListing
+from .services import search_houses, process_search_results, analyze_listings
+from urllib.parse import quote_plus
+from groq import Groq
```

Ruff (0.8.2)

`groq.Groq` imported but unused

Remove unused import: `groq.Groq`

(F401)